spec/open_api_spec.rb

```diff
@@ -45,10 +43,13 @@
     end
 
     context "customizable route prefixes" do
+      let(:path_prefix) { random_path_part }
+      let(:app_name)    { random_path }
+
       it "with a random prefix" do
         expect(ENV["PATH_PREFIX"]).not_to be_nil
         expect(ENV["APP_NAME"]).not_to be_nil
-        expect(api_v1x0_orders_url(:only_path => true)).to eq("#{URI.encode(ENV["PATH_PREFIX"])}/#{URI.encode(ENV["APP_NAME"])}/v1.0/orders")
+        expect(api_v1x0_orders_url(:only_path => true)).to eq("/#{URI.encode(ENV["PATH_PREFIX"])}/#{URI.encode(ENV["APP_NAME"])}/v1.0/orders")
```

Comment:
What introduced the extra `/` character at the beginning of the path?

Reply:
I believe it's how I'm declaring the `let` vars on lines 46, 47. The original tests called them differently.

Reply:
Got it, the original tests where here: https://github.com/ManageIQ/catalog-api/pull/291/files#diff-08bad4ad44dfcf3bf21411105657a4ceL18